### ISSUE TITLE
Select multiple contacts and have batch-actions on the selection

### ISSUE THREAD

**Issue by conspacer** [nextcloud/contacts#277]
Feature Request: Select multiple contacts (use CTRL or SHIFT etc..)
Its helpful when needing to batch edit a small subset of contacts (i.e. add/edit group, download, add/edit business details etc..)

**Comment by hitam4450** [comment]
or delete more then 1 contact at a time......

**Comment by scharc** [comment]
or export just a selected view...

**Comment by SCO-Bernhard** [comment]
or assign to groups ...

**Comment by xXSTrikeXx** [comment]
Is there already a pr? :)

**Comment by SCO-Bernhard** [comment]
Sorry, I`m from Germany and I`m not used to this shortcut: pr?

**Comment by xXSTrikeXx** [comment]
@SCO-Bernhard PR is just a programing shortcut on Github and this means, that every contributer can create a pull request which gets merged into the master code. Deutsch ist für den Deutschen an mancher Stelle echt einfacher :D But it´s an international Project 👍 

I asked @skjnldsv since he added the Label 1. to develop 👍

**Comment by skjnldsv** [comment]
No pr yet! A full rebuild of the app is scheduled, this feature will be one of the priorities :)

**Comment by rgarrigue** [comment]
@skjnldsv any ETA for the rebuild ? Or just an issues to subscribe to follow the rework's progression ?

**Comment by skjnldsv** [comment]
@rgarrigue unfortunately, no. We're slowly organising ourselves for this task, but this will require a bit of time.

**Comment by void-m4110c** [comment]
Sooo, four years have passed...may I ask, how's the self-reorganizing going? ;-)

**Comment by skjnldsv** [comment]
Very down our priority list :)

**Comment by axe312ger** [comment]
Yay, can't wait for it!

**Comment by melroy89** [comment]
> or delete more then 1 contact at a time......

yes please...!!

**Comment by miltuss** [comment]
But do something. What are you waiting for to implement this Nothing at all feature in the contacts app?

It's hell to manipulate each contact individually instead of performing a group manipulation.

There are dozens of requests for this problem all over the web!

https://help.nextcloud.com/t/mass-delete-of-contacts-how-if-possible/95410
https://help.nextcloud.com/t/help-multiple-address-books-contacts-organization/69590/2
https://help.nextcloud.com/t/contacts-multiple-contacts-selection/162099/4
https://github.com/nextcloud/contacts/issues/3054
https://github.com/nextcloud/contacts/issues/1898
...

**Comment by melroy89** [comment]
Agreed. I don't want AI. I want to be able to remove multiple contact at once.

**Comment by jgonsior** [comment]
Do you all realize that Nextcloud is open-source and free and you are welcome to implement this feature yourself? Otherwise, be thankful for having this awesome software available for free and show some gratitude! 
There are thousands of Open-Source projects being abandoned by developers due to not having a stable source of income. I'm sure the Nextcloud team is doing their best to ensure they will be continuously employed in the future, and this negative whining about a feature missing in a product that is given to you for free does not help at all.

**Comment by HibGut** [comment]
Basic functionality. It's really baffling how this hasn't been implemented from the start. 

The app has a grouping feature. A group, by definition, consists of multiple items. Am I supposed to manage them one by one? Doesn't that go hand in hand with multiple selection?

**Comment by miltuss** [comment]
When will this feature finally be available in the stable version of nextcloud?

**Comment by richardwonka** [comment]
Is there an ETA for multi-select in contacts?

**Comment by ChristophWurst** [comment]
https://github.com/nextcloud/documentation/blob/dbde0fdd992c9f5118d56d5fc334a3fe19ed93ca/user_manual/groupware/contacts.rst needs an update for this new feature.

**Comment by miltuss** [comment]
When will this feature finally be available in the Contact app? ?

**Comment by ChristophWurst** [comment]
It will be available with the next minor or major release.

**Comment by ChristophWurst** [comment]
> https://github.com/nextcloud/documentation/blob/dbde0fdd992c9f5118d56d5fc334a3fe19ed93ca/user_manual/groupware/contacts.rst needs an update for this new feature.

No documentation of this feature still: https://github.com/nextcloud/documentation/blob/f3a1403f36569802272b25c5bbfccd941c7d0b46/user_manual/groupware/contacts.rst
